.gitignore

```diff
@@ -20,4 +24,4 @@ standalonedata/
 
 worldwind.js
 
-worldwind.min.js
```

Comment:
I had the same thought, but we've been ordering .gitignore and .npmignore alphabetically. Regarding the trailing slash, I have no preference provided we're consistent.